File: 프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.js
function solution(num_list) {
    let odd_sum = 0;
    let even_sum = 0;
    num_list.forEach((n, index) => index % 2 ? odd_sum += n : even_sum += n);
    return odd_sum === even_sum ? odd_sum : odd_sum > even_sum ? odd_sum : even_sum;
}

File: 프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.py
def solution(num_list):
    odd = 0
    even = 0
    
    for idx, num in enumerate(num_list):
        if idx % 2 == 0:
            odd += num_list[idx]
        else:
            even += num_list[idx]
            
    return odd if odd > even else even